app/models/user.rb

```diff
@@ -1,37 +1,20 @@
-# == Schema Information
-#
-# Table name: users
-#
-#  id                     :integer          not null, primary key
-#  first_name             :string
-#  last_name              :string
-#  role                   :integer          default("admin")
-#  email                  :string           default(""), not null
-#  encrypted_password     :string           default(""), not null
-#  reset_password_token   :string
-#  reset_password_sent_at :datetime
-#  remember_created_at    :datetime
-#  sign_in_count          :integer          default(0), not null
-#  current_sign_in_at     :datetime
-#  last_sign_in_at        :datetime
-#  current_sign_in_ip     :inet
-#  last_sign_in_ip        :inet
-#  created_at             :datetime         not null
-#  updated_at             :datetime         not null
-#
-
 class User < ApplicationRecord
   SALE_ROLE = 'sale'.freeze
   ADMIN_ROLE = 'admin'.freeze
   MODERATOR_ROLE = 'moderator'.freeze
   ROLES = [SALE_ROLE, ADMIN_ROLE, MODERATOR_ROLE].freeze
+
   has_many :companies, dependent: :destroy
   belongs_to :tenant, optional: true
+
+  accepts_nested_attributes_for :tenant
```

Comment:
what happens if I'm sending from sing up page empty tenant name?
Admin must have tenant (add validation) for admin